tests/framework/utils.py

```diff
@@ -13,6 +13,7 @@
 import time
 import platform
 
+from typing import Dict
```

Comment:
I think since 3.7 on you can also just use dict https://stackoverflow.com/questions/37087457/difference-between-defining-typing-dict-and-dict

(not important)

Reply:
Oh cool, good to know! My python knowledge is pretty much limited to what existed in 3.6 haha